pyrometheus/__init__.py

```diff
@@ -324,6 +410,76 @@ def get_temperature(self, H_or_E, T_guess, Y, do_energy=False):
 
         return T
 
+    def get_falloff_rates(self, T, C, k_fwd):
+        k_high = np.zeros(self.num_falloff)
+        k_low = np.zeros(self.num_falloff)
```

Comment:
It's better (read: easier to read, friendlier to lazy eval) to create new data structures rather than to make "empty" ones and then fill them with the desired values.

Imagine this code being run in a setting where `T` is an array of temperatures that lives on a GPU. :)

Reply:
I've changed how I handle rate coefficients to (I think) follow this suggestion more closely.